Update interfaces/webxr.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/1110c13/whatwg/idl/webxr.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/526752636

foolip commented:
Blocked on https://github.com/web-platform-tests/wpt.fyi/issues/1276 because I'm not able to verify that the changes didn't regress something. @lukebjerring FYI.

autofoolip committed:
Update interfaces/webxr.idl

Source: https://github.com/tidoust/reffy-reports/blob/1110c13/whatwg/idl/webxr.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/526752636
